src/Pagerfanta/Adapter/ElasticsearchAdapter.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Pagerfanta\Adapter;
+
+use Elasticsearch\Client;
+
+/**
+ * ElasticsearchAdapter.
+ *
+ * Used with the official Elasticsearch PHP library.
+ * 
+ * @author Danny Sipos <danny@webomelette.com>
+ */
+class ElasticsearchAdapter implements AdapterInterface
+{
+
+  /**
+   * The Elasticsearch query parameters.
+   *
+   * @var array
+   */
+  private $params;
```

Comment:
please use 4 spaces for the indentation to match the coding standards of the library